serverless.yml

```diff
@@ -0,0 +1,19 @@
+service: nookwill
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: us-west-2
+
+package:
+ include:
+   - backend/handler.js
```

Comment:
we also don't have this folder yet but I think it's fine to leave in for now since we will as soon as we build anything on the backend